setup.cfg

```diff
@@ -1,3 +1,12 @@
 [sdist_dsc]
 epoch: 1
-debian-version: 2intermodalics
+debian-version: 3intermodalics
```

Comment:
nit: As the main version was updated from `0.2.6` to `0.2.12` in the meantime, the debian version should be reset to `1intermodalics` (according to the [Debian Policy Manual](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version)). The debian version (or revision) is usually only increased when a new release is made for the same upstream version.